modules/nf-core/sentieon/tnhaplotyper2/tests/main.nf.test

```diff
@@ -0,0 +1,378 @@
+                input[6] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/mills_and_1000G.indels.hg38.vcf.gz',checkIfExists:true)]
+                input[7] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/mills_and_1000G.indels.hg38.vcf.gz.tbi',checkIfExists:true)]
+                input[8] = true
+                input[9] = true
+                """
+            }
+        }
+        then {
+            assert process.success
+            assertAll(
+                { assert snapshot(
+                    path(process.out.vcf[0][1]).vcf.summary,
+                    file(process.out.index[0][1]).name,
+                    process.out.versions,
+                    path(process.out.versions[0]).yaml
+                    ).match() }
+            )
+        }
+    }
+
+
+    test("no pon -- stub") {
+        options '-stub'
+        when {
+            params {
+                module_args  = ''
+                module_args2 = '--tumor_sample tumour --normal_sample normal'
+                module_args3 = ''
+                module_args4 = ''
+            }
+            process {
+                """
+                input[0] = NORMAL_ADDORREPLACEREADGROUPS.out.bam
+                    .join(NORMAL_ADDORREPLACEREADGROUPS.out.bai)
+                    .join(TUMOUR_ADDORREPLACEREADGROUPS.out.bam)
+                    .join(TUMOUR_ADDORREPLACEREADGROUPS.out.bai)
+                    .map{
+                    meta,normal_bam,normal_bai,tumour_bam,tumour_bai->
+                    [[id:'test'],[normal_bam,tumour_bam],[normal_bai,tumour_bai],[]]
+                    }
+                input[1] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.dict',checkIfExists:true)]
+                input[2] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta',checkIfExists:true)]
+                input[3] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/sequence/genome.fasta.fai',checkIfExists:true)]
+                input[4] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/gnomAD.r2.1.1.vcf.gz',checkIfExists:true)]
+                input[5] = [[:],file(params.modules_testdata_base_path + 'genomics/homo_sapiens/genome/chr21/germlineresources/gnomAD.r2.1.1.vcf.gz.tbi',checkIfExists:true)]
+                input[6] = [[:],[]]
+                input[7] = [[:],[]]
+                input[8] = false
+                input[9] = false
+                """
+            }
+        }
+        then {
+            assert process.success
+            assertAll(
+                { assert snapshot(process.out).match() }
+            )
+        }
+    }
+
+
+    test("with pon -- stub") {
```

Comment:
Why do you create multiple stub test if there is no difference in the output ?

Reply:
Uhh, because the automatic conversion did it for me ;)